docs/components/ColorPicker.md

```diff
@@ -8,18 +8,10 @@ Dialog which allows the user to select color.
 ## Example
 
 ```javascript
-import React from 'react';
-import { View, ColorPicker } from 'magic-script-components';
-
-export default class MyApp extends React.Component {
-  render () {
-    return (
-      <View name='main-view'>
-        <ColorPicker color={[1, 0.5, 1, 0.8]} />
-      </View>
-    );
-  }
-}
+<ColorPicker
```

Comment:
I'd actually prefer a full example instead. Can you add back the `View` tag as well as `render` method and surrounding code?

Reply:
Will do.